### PR TITLE
Add tests for transferring ImageBitmap objects.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25864)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/352919304)

File: 2dcontext/imagebitmap/createImageBitmap-transfer.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    worker = new Worker("transfer-worker.js");
+    worker.addEventListener("message", function(event) {
+        let { name, bitmap } = event.data;
+        tests[name](bitmap);
+    });
+});
+
+for (let { name, factory } of imageSourceTypes) {
+    promise_test(function(t) {
+        let message_handler = new Promise(function(resolve) {
```
Comment (foolip) [inline 2dcontext/imagebitmap/createImageBitmap-transfer.html]: Can you inline this where it's used, with a comment pointing to what invokes resolve? Having it out of order and the indirection made me scratch my head for a bit.

File: 2dcontext/imagebitmap/createImageBitmap-transfer.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+<meta charset=utf-8>
+<title>createImageBitmap transferring test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/media.js"></script>
+<script src="/common/namespaces.js"></script>
+<script src="common.sub.js"></script>
+<div id=log></div>
+<script>
+let worker, tests = {};
```
Comment (foolip) [inline 2dcontext/imagebitmap/createImageBitmap-transfer.html]: Can you rename tests to continuations or something? It's a map from test names to callbacks, not what I thought at first.

**Review by foolip — APPROVED** [review]
LGTM % style suggestions for you to do as you please with

**Review by foolip — APPROVED** [review]
Thanks, looks great!

**Pull request merged**